### PR TITLE
Data basel

### PR DESCRIPTION
**Pull request by jburel**
Add the new set of data used for Basel

cc @pwalczysko @will-moore

### CODE REVIEW

File: data.md [hunk 1]
```diff
@@ -6,43 +6,49 @@ Data imported for users (including trainers)
...
 
 Data imported for trainers only
 -------------------------------
 
 - "INMAC384-DAPI-CM-eGFP_59223_1.xdce" plate:
-   * http://downloads.openmicroscopy.org/images/HCS/INCELL2000/INMAC384-DAPI-CM-eGFP_59223_1/
+   * https://downloads.openmicroscopy.org/images/HCS/INCELL2000/INMAC384-DAPI-CM-eGFP_59223_1/
 - "chromatine-condensation" dataset:
-   * http://downloads.openmicroscopy.org/images/TIFF/condensation/
+   * https://downloads.openmicroscopy.org/images/TIFF/condensation/
```
Comment (pwalczysko) [inline data.md]: Everybody has this one. Move to upper section

File: data.md [hunk 2]
```diff
@@ -6,43 +6,49 @@ Data imported for users (including trainers)
...
 - "Fiji Batch" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_10_R3D_D3D.dv 
-   * http://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_11_R3D_D3D.dv
+   * https://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_10_R3D_D3D.dv 
+   * https://downloads.openmicroscopy.org/images/DV/elena/P-TRE/P-TRE_11_R3D_D3D.dv
+- "FRAP" dataset:
+   * https://downloads.openmicroscopy.org/images/DV/will/FRAP
+- "Kymograph" dataset:
+   * http://jcb-dataviewer.rupress.org/jcb/browse/4573/12710/ Image identified by letter F
 - "Multi-Z timelapse" dataset:
```
Comment (pwalczysko) [inline data.md]: Change to ``dv-iain-multiTZ`` as the name of this dataset

**Comment by pwalczysko** [comment]
In the ``trainer-1 only`` section, we have one more dataset, `` dv-alexia-BoxIt``

In fact, the ``dv-alexia-BoxIt`` is https://github.com/ome/training-repos/pull/11/files#diff-494b4c66d4128fe395a8c9607fab3629L10 (alexia-cajal-bodies)

File: data.md [hunk 3]
```diff
@@ -6,43 +6,49 @@ Data imported for users (including trainers)
...
 - "siRNAi-HeLa" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
+   * https://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
 - "Cajal-bodies" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
+   * https://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
 - "CellProfiler images" dataset:
-   * http://downloads.openmicroscopy.org/images/TIFF/cellprofiler/fruit-fly-cells
+   * https://downloads.openmicroscopy.org/images/TIFF/cellprofiler/fruit-fly-cells
 - "Condensation" dataset:
```
Comment (pwalczysko) [inline data.md]: it is ``condensation`` (lower case ``c``)

File: data.md (hunk 1 above)
Comment (pwalczysko) [reply data.md]: Sorry, this is highlighted correctly. This ``chromatine-condensation`` is called ``condensation`` and everybody has it.

File: data.md [hunk 4]
```diff
@@ -6,43 +6,49 @@ Data imported for users (including trainers)
 --------------------------------------------
 
 - "siRNAi-HeLa" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
+   * https://downloads.openmicroscopy.org/images/DV/siRNAi-HeLa/
 - "Cajal-bodies" dataset:
-   * http://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
+   * https://downloads.openmicroscopy.org/images/DV/alexia/cajal-bodies
 - "CellProfiler images" dataset:
```
Comment (pwalczysko) [inline data.md]: only trainers have this one (cell profiler), move this to lower section

**Comment by jburel** [comment]
@pwalczysko updated

**Comment by jburel** [comment]
Merging so we can tag the repo

**Pull request merged**